stable/magento/templates/NOTES.txt

```diff
@@ -1,3 +1,5 @@
++{{- if or .Values.mariadb.enabled .Values.externalDatabase.host -}}
```

Comment:
I think we should improve the chart UI mainly w.r.t to the copy-paste instructions.
I've created a PR here https://github.com/tompizmor/charts/pull/1 for your consideration. Please feel free to change.

Reply:
Thanks for the PR @sameersbn. I included the changes here